wallet/src/de/schildbach/wallet/database/entity/BlockchainIdentityData.kt

```diff
@@ -100,7 +100,6 @@ data class BlockchainIdentityData(var creationState: CreationState = CreationSta
     enum class CreationState {
         NONE,   // this should always be the first value
         UPGRADING_WALLET,
-        MIXING_FUNDS,
```

Comment:
Mixing is no longer part of the Username Creation Process, but the UI hasn't been updated to reflect that.